index.html

```diff
@@ -0,0 +1,266 @@
+                            </li>
+                            <li>
+                                <a class="bottom-header-ul-li-text transition font" href="#">Join</a>
+                            </li>
+                            <li>
+                                <a class="bottom-header-ul-li-text transition font" href="#">Sponsor</a>
+                            </li>
+                            <li>
+                                <a class="bottom-header-ul-li-text transition font" href="#">News</a>
+                            </li>
+                           </ul>
+                           <button type="button" class="campaign transition font">
+                            ComdeyBurst Initiative
+                           </button>
+                        </section>
+                    </div>
+                </section>
+            </nav>
+        </div>
+        <div class="page-header-body">
+            <h1 class="header-text-head font for-phone " >"Hey! Comic Lovers"</h1>
+            <h1 class="header-text-head-desk font for-desktop " >"Hey! Comic Lovers"</h1>
+            <h2 class="header-text-body font for-phone">
+                FIND THE BEST
+                <br>
+                CONCERT WHEREVER
+                <br>
+                YOU ARE
+            </h2>
+            <h2 class="header-text-body-desk font for-desktop">
+                FIND YOUR PERFECT
+                <br>
+                COMIC WHEREVER
+                <br>
+                YOU ARE
+            </h2>
+            <section class="header-text-description-container for-phone">
+                <p class="header-text-description font">
+                    Discover the best live comic, perfect 
+                    <br>
+                    <br>
+                    to your comdey type.
+                </p>
+            </section>
+            <section class="header-text-description-container-desk for-desktop">
+                <p class="header-text-description-desk font"> Discover the best live comic, perfect to your comdey type.
+                </p>
+            </section>
+            <section class="T-D">
+                <h2 class="date" >2023.8.15(THU) ~ 16(FRI)</h2>
+                <P class="address">@Rajeeb Gandhi Stadium,Yash Waghmare & More</P>
+            </section>
+        </div>
+    </section>
+    <section id="main" > 
+        <div class="main for-phone">
+            <section>
+                <h3 class="main-head font">Top Specials</h3>
+                <div class="red-line">_____</div>
+            </section>
+            <section class="top-specials-container ">
+                <div class="top-specials-item">
+                    <img class="comic" src="resources/icons/comic-2.png" alt="A Headphone" role="img">
+                    <h3 class="number font">Kaksha Gyarvi</h3>
+                    <section class="special-info-div">
+                        <p>
```

Comment:
- [ ] Based on the feedback from the previous reviewer, it would be great if you could ensure that the mobile design closely resembles the provided [Behance template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in spacing, margins, etc. While using different images is acceptable, it's important to maintain consistent alignment. Currently, I noticed that the image is not properly aligned—it appears to have more space than the text. Please take a moment to review and adjust accordingly. Thank you for your attention to detail! 😊👍

   ### Your design:
   ![image](https://github.com/Shahid-Bagwan/ComedyBurst/assets/107952724/7bbd9aaa-3aa6-4bb2-9161-6090e709c581)

   ### Figma design:
   ![image](https://github.com/Shahid-Bagwan/ComedyBurst/assets/107952724/ff0d4584-bc31-409a-8d3a-701ec61fc96c)